### ISSUE TITLE
Allele note is erased when background is added

### ISSUE THREAD

**Issue by jseager7** [pombase/canto#2079]
Blocking #2067 

Adding or editing the background text of an allele with a note on it seems to cause that note to disappear. You can safely add or edit a note if the background is already there, but any change to the background text seems to reset the note field.

I haven't checked if the note gets removed from the database or not, but there are no errors when creating another note for the same allele so I suspect the database is fine.

**Comment by kimrutherford** [comment]
Hi James.  I haven't been able to reproduce this bug.  Perhaps it's caused by the same bug as #2077 but it doesn't seem like it.  Could you test again after applying the fix from #2077?  Thanks.

**Comment by jseager7** [comment]
I've done some further testing, and the bug persists after applying the quick fix mentioned in https://github.com/pombase/canto/issues/2077#issuecomment-543474082. However, I've noticed that this bug _doesn't_ happen when the single allele is part of a multi-allele genotype. So if you want to try reproducing this again, here's the steps I followed:

1. Start an entirely new curation session (or just delete all the genotypes in a session);
2. add exactly one single allele genotype (I did a quick deletion);
3. add a note on that genotype;
4. add a background on that genotype;
5. check if the note has disappeared.
